### PR TITLE
chore: modernize linting

### PR DESCRIPTION
**Pull request by abalabahaha**
All autofix changes are isolated in the last commit for `git blame`. Don't squash.

Supersedes #1531.

### CODE REVIEW

File: lib/Constants.d.ts
Comment (bsian03) [inline lib/Constants.d.ts:1]: Should really add `unknown` comments in audit log actions so constants value line up with each other

File: lib/voice/Piper.js [hunk 1]
```diff
@@ -105,7 +105,8 @@ class Piper extends EventEmitter {
             this.streams.push(source.once("error", (e) => this.stop(e)));
         }
 
-        if(options.format === "opusPackets") { // eslint-disable no-empty
+        // eslint-disable-next-line no-empty
+        if(options.format === "opusPackets") {
```
Comment (bsian03) [inline lib/voice/Piper.js]: Can probably remove this if it isn't being used? It isn't in the options list nor is it referenced anywhere else

Comment (bsian03) [reply lib/voice/Piper.js]: Not outdated, still relevant https://github.com/abalabahaha/eris/blob/8e8569ae40a299dcdea52496b65329398d29bc46/lib/voice/Piper.js#L109

Comment (babiabeo) [reply lib/voice/Piper.js]: Using `switch case` would be better

Comment (bsian03) [reply lib/voice/Piper.js]: would be out of scope with this PR, but idk if @abalabahaha wants to change it here or leave it for another PR

Comment (abalabahaha) [reply lib/voice/Piper.js]: Out of scope for a linting PR, but it's useful and doesn't cost much to keep around so should probably fix docs

File: lib/Constants.d.ts
Comment (abalabahaha) [reply lib/Constants.d.ts:1]: It's a fairly long list and there are actual decimal number ranges (unlike the bitfields), so it seems better to not have `APPLICATION_COMMAND_PERMISSION_UPDATE` add 25 spaces to `GUILD_UPDATE` from 60 lines away.

**Pull request merged**